test/integration/hooks/associations.test.js

```diff
@@ -62,14 +63,14 @@ describe(Support.getTestDialectTeaser('Hooks'), function() {
           , beforeHook = false
           , afterHook = false;
 
-          this.Tasks.beforeUpdate(function(task, options, fn) {
+          this.Tasks.beforeUpdate(function(task, options) {
             beforeHook = true;
-            fn();
+            return Promise.resolve();
```

Comment:
Oh, [Like this :wink: ](https://www.google.co.in/search?client=opera&q=recursion&sourceid=opera&ie=UTF-8&oe=UTF-8) (may have some spelling mistakes)